modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/test/FilterContextProviderTest.java

```diff
@@ -96,12 +98,16 @@ public void testCreateContext() throws Exception {
 				clazz.getMethod(MockResource.METHOD_NAME, String.class),
 				_mockResource));
 
-		Assert.assertTrue(filter instanceof TermFilter);
+		Assert.assertTrue(filter instanceof QueryFilter);
```

Comment:
I feel we can revisit in the future to find another way to test the FilterContextProvider more decoupled from the implementation